rest-api/test/unit_test/api_test/questionnaire_response_api_test.py

```diff
@@ -87,6 +87,11 @@ def test_insert(self):
     get_response = self.send_get(_questionnaire_response_url(participant_id) + "/" + response['id'])
     self.assertJsonResponseMatches(resource, get_response)
 
+    # verify the primary language data is stored in participant summary
+    ps = self.send_get('Participant/%s/Summary' % participant_id)
+    primary_language = resource.get('extension',[])
+    self.assertEquals(primary_language[0]['valueCode'], ps['primaryLanguage'])
```

Comment:
Rather than asserting they are equal, can you assert the expected value? Is the questionnaire response above actually setting this?